lib/routes/bilibili/bangumi.ts

```diff
@@ -1,66 +1,68 @@
-import { Route, ViewType } from '@/types';
-import got from '@/utils/got';
+import { Data, DataItem, Route, ViewType } from '@/types';
+import cache from '@/utils/cache';
+import { EpisodeResult } from './types';
+import utils from './utils';
 
 export const route: Route = {
-    path: '/bangumi/media/:mediaid',
+    path: '/bangumi/media/:mediaid/:embed?',
     name: '番剧',
+    maintainers: ['DIYgod', 'nuomi1'],
+    handler,
+    example: '/bilibili/bangumi/media/9192',
```

Comment:
Please keep these properties in their original order. Do not perform unnecessary reordering.

Reply:
Done.